ansible/all-in-one/roles/xsnippet/templates/http.js.j2

```diff
@@ -0,0 +1,22 @@
+function get_raw_snippet(request) {
+    var regex = /^\/([A-Za-z0-9]+)\/raw\/?$/;
```

Comment:
nit: It'd be nice to see `([a-zA-Z0-9])` moved to some const pattern just to increase readability. Although, it may require changing a code a bit: using string formatting and `RegEx` class explicitly. So maybe it's something we can do later on.

Reply:
Sure, we can give it a try! The only caveat is that not everything is supported in this JS engine. E.g. you can't use `const`s.